apps/docs/stories/utils/base-args.ts

```diff
@@ -14,7 +14,7 @@ const args: ArgValues = {
 const argTypes: ArgTypes = {
   'auth-token': {
     type: 'string',
-    description: 'Auth token `string`',
+    description: 'Auth token `string`, the web component token you create via API. See Authorization below',
```

Comment:
Is this generic and will also show up for payment, card and bank account form? In that case we would have to show an alternative auth token text.
I'd say let's just say `'Auth token `string`, see Authorization below' which would cover every use case.